### PR TITLE
Remove source map for production files

### PR DESCRIPTION
**Pull request by benoitchantre**
## Description

Addresses issue #305 by using the environment to determine wether to include or not a source map.

## List of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [ ] This change has been added to CHANGELOG.md
- [x] I want my code added to WP Rig.

### CODE REVIEW

File: gulp/scripts.js [hunk 1]
```diff
@@ -16,8 +16,10 @@ export default function scripts(done) {
 	// Get a fresh copy of the config
 	const config = getThemeConfig(true);
 
+	const useSourceMap = !isProd;
```
Comment (ataylorme) [inline gulp/scripts.js]: Please just use ‘!isProd’ in the source map calls instead of creating a new variable.

Comment (benoitchantre) [reply gulp/scripts.js]: Done. This was my first attempt. I probably did a mistake checking the files. Thanks for the review.

**Comment by mor10** [comment]
@ataylorme can you milestone this?

**Comment by ataylorme** [comment]
@mor10 added to the `2.0` milestone. Waiting on one more +1 before merging

**Pull request merged**